src/handbook/company/security/human-resources.md

```diff
@@ -0,0 +1,120 @@
+include an assessment of job performance, competence in the role, adherence to
+company policies and code of conduct, and achievement of role-specific objectives.
+
+
+### Terms & Conditions of Employment
+
+Company policies and information security roles and responsibilities shall be
+communicated to employees and third-parties at the time of hire or engagement,
+and employees and contractors are required to formally acknowledge their
+understanding and acceptance of their security responsibilities.
+
+Employees and relevant third-parties shall follow all FlowForge information
+security policies.
+
+### Management Responsibilities
+
+Management shall be responsible for ensuring that information security policies
+and procedures are reviewed annually, distributed and available, and that
+employees and contractors abide by those policies and procedures for the duration
+of their employment or engagement.
+
+Annual policy review shall include a review of any linked or referenced procedures,
+standards or guidelines.
+
+Management shall ensure that information security responsibilities are
+communicated to individuals, through written job descriptions, policies or some
+other documented method which is accurately updated and maintained.
+
+Compliance with information security policies and procedures and fulfillment of
+information security responsibilities shall be evaluated as part of the performance
+review process wherever applicable.
+
+Management shall consider excessive pressures, and opportunities for fraud when
+establishing incentives and segregating roles, responsibilities, and authorities.
+
+
+### Information Security Awareness, Education & Training
+
+All FlowForge employees and third-parties with administrative or privileged
+technical access to FlowForge production systems and networks shall complete
+security awareness training at the time of hire and annually thereafter.
+
+Management shall monitor training completion and shall take appropriate steps to
+ensure compliance with this policy. Employees and contractors shall be aware of
+relevant information security and data privacy policies and procedures. The
+company shall ensure that personnel receive security and data privacy training
+appropriate to their role and data handling responsibilities.
+
+In order to maintain a robust level of security awareness, the company will
+provide security-related updates and communications to company personnel on an
+on-going basis through multiple communication channels as needed.
+
+### Termination Process
+
+Employee and contractor termination and offboarding processes shall ensure that
+physical and logical access is promptly revoked in accordance with company SLAs
+and policies, and that all company issued equipment is returned.
+
+Any security or confidentiality agreements which remain valid after termination
+shall be communicated to the employee or contractor at time of termination.
```

Comment:
Please make sure a checklist is created/updated with this item @Tracy-Anthony @knolleary

Reply:
done via - https://github.com/flowforge/admin/pull/186